app/policies/planning_application_policy.rb

```diff
@@ -27,7 +30,7 @@ def update?
   end
 
   def unpermitted_statuses
```

Comment:
I think this manual checking of `permitted_statuses` is now handled by AASM, unless I'm misreading what it is doing.

Reply:
I've removed this - but left the permitted_status method for now as this relates to user privilege.